_projects/2024/100535241/100535241.Rmd

```diff
@@ -0,0 +1,824 @@
+
+```{r original-plot, echo=FALSE, fig.cap="Original graph from ABC article", layout="l-body-outset"}
+knitr::include_graphics("~/Documents/UC3M - coursework/Data Vizualisation/dataviz/ABC_tax_income_plot.png")
+```
+
+The graph shows the median income of Australians by occupation compared to who would benefit from proposed tax changes. The graph highlights those who will benefit under the changes and those who will be worse off. To distinguish the two groups, the authors use a horizontal line from the y-axis at the benefit cutoff point, \$146,486, and background shading.
+
+The purpose of the graph is to highlight that most people are expected to be better off under the changes. Instead of estimating the number of people in raw terms that would benefit, the plot uses disagregated tax data by occupation and gender to display which groups are better and worse off.
+
+The graph is effective because you can easily conclude that the majority of people will be better off under the changes, regardless of sex or occupation. The use of a linear y-axis to show income is effective in isolating the very high income occupations which are outliers to the data. It creates a white space which contrasts against the concentrated grouping of occupations below the cutoff point.
+
+# Setup
+
+Load packages, set global and local parameters, and read in our dataset.
+
+### Load required packages
+
+The packages are for data cleaning, graph generation, adding themes and organising subplots into the final graph.
+
+```{r packages, message=FALSE, warning=FALSE}
+
+library(tidyverse)
+library(readxl)
+library(patchwork)
+library(grid)
+library(magick)
+library(ggbeeswarm)
+library(cowplot)
+library(showtext) # most similar font found but couldn't match exact
+sysfonts::font_add_google("Hind", family="hind")
+showtext_auto()
+library(geomtextpath)
+
+```
+
+```{r parameters, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, 
+                      fig.align="center", 
+                      fig.showtext=TRUE)
+```
+
+### Load data in from web
+
+Now we load the data from the `data.gov.au` page. The data comes from Table 15B of the 2020-21 Australian Tax Office individual statistics.
+
+This is some of the most granular tax data available. This choice of data allows the authors to create a graph with these dimensions:
+
+-   Occupation
+
+-   Number of employees in the occupation (shown as % of total employees on the x-axis)
+
+-   Median income of each occupation
+
+-   Splits by gender
+
+```{r raw_data}
+library(httr)
+url <- "https://data.gov.au/data/dataset/07b51b39-254a-4177-8b4c-497f17eddb80/resource/1535c9eb-8385-41af-ae65-f71b2be77d6d/download/ts21individual15occupationsex.xlsx"
+response <- GET(url)
+file_path <- tempfile(fileext = ".xlsx")
+writeBin(content(response, "raw"), file_path)
+
+data <- read_excel(file_path,
+                   sheet="Table 15B",
+                   skip=1) # Initial clean step to remove unused row
+
```

Comment:
This is fine, but if the resource doesn't exist in the future, your post cannot be replicated. So in the same way as with the original plot above, just include the file in your PR close to this Rmd, and then just `data <- read_excel("ts21individual15occupationsex.xlsx")`.